srcpkgs/hikari/template

```diff
@@ -33,5 +36,9 @@ pre_build() {
 }
 
 post_install() {
+	if [ "$build_option_setuid" ]; then
+		# must be setuid without logind!
```

Comment:
This comment doesn't make sense, seatd exists as well.

Reply:
Should be fixed in all instances.